deps/chakrashim/src/inspector/v8-inspector-session-impl.cc

```diff
@@ -77,6 +82,11 @@ V8InspectorSessionImpl::V8InspectorSessionImpl(V8InspectorImpl* inspector,
       this, this, agentState(protocol::Schema::Metainfo::domainName)));
   protocol::Schema::Dispatcher::wire(&m_dispatcher, m_schemaAgent.get());
 
+  m_timeTravelAgent = wrapUnique(new V8TimeTravelAgentImpl(
+      this, this, agentState(protocol::TimeTravel::Metainfo::domainName)));
+  protocol::TimeTravel::Dispatcher::wire(&m_dispatcher,
+                                         m_timeTravelAgent.get());
```

Comment:
I thought about it and decided not to mess with the dispatcher dynamically.  The TTD dispatcher will be wired up, but any attempts to use it will return an error.